File: src/test/unit/LectureIDTest.java
package test.unit;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

import IDs.InstructorID;
import IDs.LectureID;

public class LectureIDTest {
	
	@Test
	public void test_InstructorID_default_constructor() {
		
		LectureID testLectureID = new LectureID("test");
		
		testLectureID.setID("test");
		assertEquals("test", testLectureID.getID());
		
	}

}

File: Iteration 3/pythonCode/LectureIDTest.py
#!/usr/bin/env python
from LectureID import LectureID
import unittest
class LectureIDTest(unittest.TestCase):
    def test_LectureIDTest_ID(self):
        # generated source for method test_LectureIDTest_ID 
        testLectureID = LectureID("test")
        testLectureID.setID("test")
        self.assertEqual("test", testLectureID.getID())
if __name__ == '__main__':
    unittest.main()